AKS (PRIMES is in P)/cl3.md

```diff
@@ -0,0 +1,35 @@
++++
+reference = "Part for Claim 3 from ~ https://www3.nd.edu/%7eandyp/notes/AKS.pdf"
+state = "Draft"
+statement = """
+cl3a.1 $e |- G = ( Z/nZ ` R ) $.
+cl3a.2 $e |- O = ( od ` G ) $.
+cl3a.3 $e |- F = ( GF_oo ` P ) $. // F is the algebraic closure of F_p
+cl3a.4 $e |- A = ( ( ZRHom ` P ) ` a ) $. // ` A ` is the image of a by the canonical homomorphism from the integers to the ring ` F `
+cl3a.5 $e |- .^ = ( .g ` ( mulGrp ` F ) ) $.
+cl3a.6 $e |- .1. = ( 1r ` F ) $.
+cl3a.7 $e |- +. = ( +g ` F ) $.
+cl3a.8 $e |- .0. = ( 0g ` F ) $.
+cl3a.9 $e |- ( ph -> N e. ( ZZ>= ` 2 ) ) $.
+cl3a.10 $e |- ( ph -> P e. Prime ) $.
+cl3a.11 $e |- ( ph -> P || N ) $.
+cl3a.12 $e |- ( ph -> R e. ( ZZ>= ` 2 ) ) $.
+cl3a.13 $e |- ( ph -> ( N gcd R ) = 1 ) $.
+cl3a.14 $e |- ( ph -> ( ( 2 logb N ) ^ 2 ) < ( O ` N ) ) $.
+cl3a.15 $e |- ( E = { x e. ZZ | ( K e. NN0 /\ J e. NN0 /\
+( ( ( N / P ) ^ K ) x. ( P ^ J ) ) ) } ) $.
```

Comment:
I think you can write this set as:
```
ran ( k e. NN0 , j e. NN0 |-> ( ( ( N / P ) ^ k ) x. ( P ^ j ) ) ) )
```

Reply:
Ah nevermind, I didn't see that p divides N, yeah, that works. I assumed that N was arbitrary, then $\frac{N}{p}$ could be in $\mathbb{Q}-\mathbb{Z}$ justifying my version. Since I missed that it is clear that $\frac{N}{p}$ is an integer. I will fix the description.

Reply:
Also I think we should tackle this claim first. It is small and it should be straightforward to prove.